wisp-task/src/main/kotlin/wisp/task/RepeatedTask.kt

```diff
@@ -94,19 +93,28 @@ class RepeatedTask(
     return runBlocking {
       var status = Status.OK
       retry(retryPolicy) {
-        // TODO: metrics
-        val startTime = clock.instant()
-        val timedResult = measureTimeMillis {
-          status = task(name, taskConfig)
+        var timedResult = 0L
+        try {
+          timedResult = measureTimeMillis {
+            status = task(name, taskConfig)
+          }
+        } catch (e: Exception) {
+          repeatedTaskMetrics.retryCount.increment()
```

Comment:
Is this the same semantics as before? I'd think that retryCount increments at the start of a retry.

Reply:
There was no retry metric count before.

Reply:
It should be a failed count then.